lib/stores.py

```diff
@@ -0,0 +1,66 @@
+# This file is part of Cockpit.
+#
+# Copyright (C) 2017 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+# Shared GitHub code. When run as a script, we print out info about
+# our GitHub interacition.
+
+import os
+import socket
+import urllib
+
+from lib.constants import IMAGES_DIR
+
+# Cockpit image/log server CA
+CA_PEM = os.getenv("COCKPIT_CA_PEM", os.path.join(IMAGES_DIR, "files", "ca.pem"))
+
+# Servers which have public images
+PUBLIC_STORES = [
+    "https://images-frontdoor.apps.ocp.ci.centos.org/",
+    "https://images-cockpit.apps.ci.centos.org/",
```

Comment:
While you are at it, please add a commit that drops this. The OCP3 cluster got shut down, and trying to ping this store hangs, and is just in vain.